src/components/App.tsx

```diff
@@ -3381,13 +3385,9 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     );
     if (elementClickedInside) {
       const elementCenterX =
-        elementClickedInside.x +
-        elementClickedInside.width / 2 +
-        this.parentDOMLeft;
```

Comment:
What text jumping? Can you post steps to reproduce your case?

Either way, we certainly cannot offset coords here because they're already in scene coords and offsetting this breaks the element-center detection.

I can reproduce one jumping, when zoomed, in a non-(0,0) canvas offset, but that's unrelated to the above code, and offsetting here doesn't fix it (actually makes it worse).

Reply:
@dwelle you can apply some `width, height, top and left` to the wrapper where excalidraw is getting rendered, this will produce a scenario where host app has some left, top, etc and then try to add a text element / edit 

<img width="865" alt="Screenshot 2020-07-27 at 1 52 36 PM" src="https://user-images.githubusercontent.com/11256141/88520030-a5ec4380-d010-11ea-8697-d087726a4d1a.png">

Reply:
I'm doing that. Apart from the zoom bug (which will require a separate fix) I mentioned above, I can't reproduce anything. Can you provide steps?